pages/osx/pkgutil.md

```diff
@@ -0,0 +1,15 @@
+# pkgutil
+
+> Query and manipulate Mac OS X Installer packages and receipts.
```

Comment:
Is the program's name really Mac OS X Installer? I can't find the name anywhere.

Reply:
Yes. ![image](https://user-images.githubusercontent.com/145113/72368578-5b3a5800-36cc-11ea-82fc-cdb201cc2b3b.png)